plugins/woocommerce/tests/e2e-pw/README.md

```diff
@@ -19,6 +19,8 @@ This is the documentation for the new e2e testing setup based on Playwright and
 
 ## Pre-requisites
 
+In case this is the first time building WooCommerce, it is advised to go through the [WooCommerce Monorepo prerequisites]([url](https://github.com/woocommerce/woocommerce/blob/trunk/README.md#prerequisites)) first.
```

Comment:
The URL is not rendered in GitHub, probably because the's an extra [url] in there between the parentheses.

Reply:
Nice catch, I'm updating this. Thank you!